app/javascript/hyacinth_ui_v1/components/digital_objects/children/Children.js

```diff
@@ -1,39 +1,61 @@
 import React from 'react';
-import produce from 'immer';
+import PropTypes from 'prop-types';
+import { Card } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+import { useQuery } from '@apollo/react-hooks';
+import { gql } from 'apollo-boost';
 
 import digitalObjectInterface from '../digitalObjectInterface';
+import AssetNew from '../new/AssetNew';
 import TabHeading from '../../ui/tabs/TabHeading';
+import GraphQLErrors from '../../ui/GraphQLErrors';
 
-class Children extends React.Component {
-  // state = {
-  //   digitalObject: {},
-  // }
-
-  // componentDidMount() {
-  //   const { match: { params: { id } } } = this.props;
-  //
-  //   digitalObject.get(id)
-  //     .then((res) => {
-  //       console.log(res.data);
-  //       this.setState(produce((draft) => {
-  //         draft.digitalObject = res.data.digitalObject;
-  //       }));
-  //     });
-  // }
-
-  render() {
-    const { data: { uid } } = this.props;
-
-    return (
-      <>
-        <TabHeading>Manage Child Assets</TabHeading>
-        <p>Child Digital Objects</p>
-        <p># Total</p>
-
-
-      </>
-    );
+const getChildAssetsQuery = gql`
```

Comment:
After rebasing, could be good to put this in digitalObject.js with the other digital object queries.

Reply:
it is done, with a new query response type that dereferences the child objects for display and the parent for the interface.

Reply:
Great, thanks!